cli/tests/activate.bats

```diff
@@ -2624,9 +2680,10 @@ EOF
 @test "nested interactive activate fails" {
   project_setup
   run bash <(cat <<'EOF'
+    _expect="$(command -v expect)"
     eval "$("$FLOX_BIN" activate)"
 
-    expect "$TESTS_DIR/activate/activate.exp" "$PROJECT_DIR"
+    $_expect "$TESTS_DIR/activate/activate.exp" "$PROJECT_DIR"
```

Comment:
question: What's wrong with expect from the PATH?

Reply:
It gets removed from `PATH` after the in-place activation before this line. Related to:

- a37bf1598e6013d17473c63bc70040df8dde687f
- https://github.com/flox/flox/pull/2552#discussion_r1900758571

Reply:
Thanks Dan, resolving conversation.